api/src/pcapi/core/users/models.py

```diff
@@ -413,6 +420,28 @@ def suspension_date(self) -> Optional[datetime]:
             return self.suspension_history[-1].eventDate
         return None
 
+    @property
+    def account_state(self) -> AccountState:
+        if self.isActive:
+            return AccountState.ACTIVE
+
+        if not self.isActive and self.suspension_history:
```

Comment:
(détail) `not self.isActive` est forcément vrai, sinon on serait déjà sorti de la fonction avec le "if" précédent, non ?